hangupsbot/plugins/restrictedadd.py

```diff
@@ -107,9 +126,19 @@ def _verify_botkeeper_presence(bot, event, command):
     if not botkeeper:
         logger.warning("no botkeeper in {}".format(event.conv_id))
 
-        yield from bot.coro_send_message(
-            event.conv,
-            _("<i>There is no botkeeper in here. I have to go...</i>"))
+        notify = bot.config.get_option('botkeeper_check_notify')
+        if notify is True:
+            yield from bot.coro_send_to_user_and_conversation(
+                _botkeeper_list(bot, event.conv_id)[0], event.conv_id,
```

Comment:
Taking the first element from the botkepper list seems flimsy to me. Would it be better to allow admins/botkeepers the ability to opt-in to receiving the notification, rather than send it to the first botkeeper in the list?

This would involve changing the design a bit, but I think it's a better outcome.